script/generate-e2e-tests.js

```diff
@@ -1090,6 +1090,10 @@ const components = new Map([
           id: 'components-select-features--with-option-groups',
           name: 'With Option Groups',
         },
+        {
```

Comment:
Is there something else I need to do besides updating this script? I'm not seeing any new snapshots get added.

Reply:
@mperrotti this file has really gotten old so one thing could be to update the test file directly to use the new array pattern we've been doing: https://github.com/primer/react/blob/main/e2e/components/Link.test.ts#L5 and updating it there 👀